docs/user-manuals/advancedstatefulset.md

```diff
@@ -142,6 +142,27 @@ Then controller will delete Pod-3 and create Pod-5 (existing Pods will be `[0,2,
 - If you just want to delete Pod-3, you should append `3` into `reserveOrdinals` list and set `replicas` to `3`.
 Then controller will delete Pod-3 (existing Pods will be `[0,2,4]`).
 
+### Specify Pod Deletion
+
+**FEATURE STATE:** Kruise v1.5.5, Kruise v1.6.4, Kruise v1.7.2
+
+Compared to manually deleting a Pod directly, using the `apps.kruise.io/specified-delete: true` specified Pod deletion will be protected by the `maxUnavailable` of the Advanced StatefulSet during deletion, and it will trigger the `PreparingDelete` lifecycle hook (see below).
```

Comment:
using the `apps.kruise.io/specified-delete: true` specified Pod deletion ->
pod deletion by labeling pod with `apps.kruise.io/specified-delete: true`